apps/cookbook/src/app/examples/data-table-example/table_example_data.ts

```diff
@@ -0,0 +1,152 @@
+export const table_example_data = [
```

Comment:
I really like the idea, but for simplicity we should consider to just reuse the data from the 'base list' component for this. Possibly just a quick refactor of that so it is not owned by the list component, but can be used here too.